lib/util.js

```diff
@@ -223,3 +224,18 @@ exports.parseFormats = function(info, debug) {
   formats.sort(exports.sortFormats);
   return formats;
 };
+
+/**
+ * Get video description from html
+ *
+ * @param {String} html
+ * @return {String}
+ */
+ exports.getVideoDescription = function(html) {
+   var regexString = '<p.*?id="eow-description".*?>(.+?)<\/p>'
+                   + '[\\n\\r\\s]*?<\/div>';
```

Comment:
why not make this a compile time regex object? rather than using the dynamic RegExp class.